app/src/main/java/com/randomappsinc/studentpicker/utils/SpeechUtil.java

```diff
@@ -0,0 +1,38 @@
+package com.randomappsinc.studentpicker.utils;
+
+import android.content.Context;
+import android.content.Intent;
+import android.speech.RecognizerIntent;
+
+import androidx.annotation.Nullable;
+
+import com.randomappsinc.studentpicker.R;
+
+import java.util.List;
+import java.util.Locale;
+
+import static android.app.Activity.RESULT_OK;
+
+public class SpeechUtil {
+    public static Intent openSpeechToTextDialog(Context context) {
```

Comment:
We should update the name to match the new logic.
```suggestion
    public static Intent getSpeechToTextIntent(Context context) {
```